.releaserc.js

```diff
@@ -51,7 +51,6 @@ const releaseConfig = {
             {
                 assets: [
                     'package.json',
-                    'package-lock.json',
```

Comment:
This is a bit of a shot in the dark since I'm not familiar with this config, but should this be replaced with `yarn.lock`?

Reply:
I think the only reason this was here before was that `package-lock.json` actually contained the version for some reason. `yarn.lock` does not, so I don't believe we need it here.